container/images/ruby/Dockerfile

```diff
@@ -0,0 +1,63 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG RUBY_VERSION
+ENV RUBY_VERSION="2.2.2"
+
+RUN /bin/bash -l -c "curl -sL https://deb.nodesource.com/setup_8.x | bash -"
+RUN /bin/bash -l -c "curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -"
+RUN /bin/bash -l -c 'echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list'
+
+RUN apt-get update && apt-get install -qy \
+    autoconf \
+    automake \
+    bison \
+    build-essential \
+    curl \
+    g++ \
+    gawk \
+    gcc \
+    git-core \
+    libc6-dev \
+    libcurl4-openssl-dev \
+    libffi-dev \
+    libgdbm-dev \
+    libncurses5-dev \
+    libreadline-dev \
+    libreadline6-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    libssl1.0.0 \
+    libtool \
+    libxslt1-dev \
+    libyaml-dev \
+    libxml2-dev \
+    make \
+    nodejs \
+    openssl \
+    patch \
+    pkg-config \
+    software-properties-common \
+    python-software-properties \
+    sqlite3 \
+    yarn  \
+    zlib1g-dev \
+    --no-install-recommends && rm -r /var/lib/apt/lists/*
+
+RUN /bin/bash -l -c "echo progress-bar >> ~/.curlrc"
+RUN curl -sSL https://rvm.io/mpapis.asc | gpg --import -
+RUN echo "Downloading and installing 'rvm'" && curl -sSL https://get.rvm.io | bash -s stable
+
+RUN echo "Downloading and installing 'rbenv'" && git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+RUN echo "export PATH=$HOME/.rbenv/bin:$PATH" >> ~/.bashrc
+RUN echo 'eval "$(rbenv init -)"' >> $HOME/.bashrc
+RUN git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+RUN echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.bashrc
+RUN echo 'export PATH="$HOME/.rbenv/versions/${RUBY_VERSION}/bin:$PATH"' >> ~/.bashrc
+
+RUN /bin/bash -l -c "echo Downloading and installing \'ruby via rbenv\'"
```

Comment:
Is there an easier way to install Ruby? Something like using Apt-get? 
It feels a bit too complicated.

Reply:
I agree its complicated and I had trouble getting 2.2.2 to work as I mentioned above, I will try another route but this was a bit fiddley

Reply:
Have a look at the new version of the changes